web/packages/teleterm/build_resources/linux/apparmor-profile

```diff
@@ -0,0 +1,9 @@
+abi <abi/4.0>,
```

Comment:
There doesn't appear to be a way to [write a backwards-compatible apparmor profile](https://askubuntu.com/questions/1517272/writing-a-backwards-compatible-apparmor-profile).

The answer to that Ask Ubuntu question links to an example from the Ubuntu Pro Client which uses a jinja template and modifies the profile based on which Ubuntu version is running. I don't think we can do that since our app isn't meant for Ubuntu only.

What we could do is first check if the given version of apparmor_parser can handle our profile (maybe with [`-preprocess` or `--skip-kernel-load --debug`](https://wiki.debian.org/AppArmor/HowToUse#Dumping_profiles)?) and only then copy it to apparmor.d and load it.

If apparmor_parser runs into an error when checking the profile, we don't copy the profile and emit a warning. This should let users on Ubuntu 22.04 use Connect. If there's another breaking change to AppArmor in the future, we'll notice that by the app not working due to a lack of profile.

How does that sound @gzdunek?

Reply:
I think testing this particular conditional branch in a VM should be enough. I already set up 20.04 myself.